components/Main/Main.jsx

```diff
@@ -1,8 +1,8 @@
 const React = require('react')
-const Filter = require('./filter')
-const TaskList = require('./task-list')
+const Filter = require('../Filter')
+const TaskList = require('../TaskList')
 
-class App extends React.Component {
+class Main extends React.Component {
```

Comment:
I avoid using Main in electron apps because of the main.js file. It always confused me when there were multiple mains.